convert_hf_to_gguf.py

```diff
@@ -525,6 +525,13 @@ def get_vocab_base(self) -> tuple[list[str], list[int], str]:
             else:
                 token: str = reverse_vocab[i]
                 if token in added_vocab:
+                    # We need to manually encode and decode the added tokens in case special characters
+                    # used for `\n` / `\t` have been manually added in the added tokens
+                    # To avoid unexpected issues - we make sure to encode single-char tokens
+                    if len(token) == 1:
+                        logger.info("Ecode-Decode special characters using AutoTokenizer")
```

Comment:
I was thinking about comparing the token before and after the encoding and print the log only if there is a difference.

Reply:
that's a good idea. Done!